package/MDAnalysis/lib/_distopia.py

```diff
@@ -77,8 +81,69 @@ def calc_bond_distance(
 def calc_bond_distance_triclinic(
     coords1: np.ndarray, coords2: np.ndarray, box: np.ndarray, results: np.ndarray
 ) -> None:
-    # redirect to serial backend
-    warnings.warn(
-        "distopia does not support triclinic boxes, using serial backend instead."
-    )
-    _calc_bond_distance_triclinic_serial(coords1, coords2, box, results)
+    distopia.calc_bonds_triclinic(coords1, coords2, box, results=results)
+
+
+def calc_angle(
+    coords1: np.ndarray, coords2: np.ndarray, coords3: np.ndarray, results: np.ndarray
+) -> None:
+    distopia.calc_angles_no_box(coords1, coords2, coords3, results=results)
+
+def calc_angle_ortho(
+    coords1: np.ndarray, coords2: np.ndarray, coords3: np.ndarray, box: np.ndarray, results: np.ndarray
+) -> None:
+    distopia.calc_angles_ortho(coords1, coords2, coords3, box[:3], results=results)
+
+
+def calc_angle_triclinic(
+    coords1: np.ndarray, coords2: np.ndarray, coords3: np.ndarray, box: np.ndarray, results: np.ndarray
+) -> None:
+
+    distopia.calc_angles_triclinic(coords1, coords2, coords3, box, results=results)
+
+
+def calc_dihedral(
+    coords1: np.ndarray, coords2: np.ndarray, coords3: np.ndarray, coords4: np.ndarray, results: np.ndarray
+) -> None:
+    distopia.calc_dihedrals_no_box(coords1, coords2, coords3, coords4, results=results)
+
+def calc_dihedral_ortho(
+    coords1: np.ndarray, coords2: np.ndarray, coords3: np.ndarray, coords4: np.ndarray, box: np.ndarray, results: np.ndarray
+) -> None:
+    distopia.calc_dihedrals_ortho(coords1, coords2, coords3, coords4, box[:3], results=results)
+
+def calc_dihedral_triclinic(
+    coords1: np.ndarray, coords2: np.ndarray, coords3: np.ndarray, coords4: np.ndarray, box: np.ndarray, results: np.ndarray
+) -> None:
+    distopia.calc_dihedrals_triclinic(coords1, coords2, coords3, coords4, box, results=results)
+
+def calc_distance_array(
+    coords1: np.ndarray, coords2: np.ndarray, results: np.ndarray
+) -> None:
+    distopia.calc_distance_array_no_box(coords1, coords2, results=results)
+
+def calc_distance_array_ortho(
```

Comment:
It's awesome that we can now use distopia for distance_array!! 🎉 

The docs for self_distance_array and distance_array need to be updated
- add distopia to backends
- add a versionchanged 2.8.0 that states that distopia is now available: it's likely a big performance boost so we should make clear which versions support this improvement